parser/list.go

```diff
@@ -164,6 +164,8 @@ func (b *listParser) Continue(node ast.Node, reader text.Reader, pc Context) Sta
 		if node.ChildCount() == 1 && node.LastChild().ChildCount() == 0 {
 			return Close
 		}
+
+		reader.Advance(len(line)-1)
```

Comment:
nit: A comment explaining "why" would be helpful here

Reply:
Hm I already forgot :D it would get me a while to get into this

Reply:
😆 That's why I write the "why" comments